pinot-plugins/pinot-minion-tasks/pinot-minion-builtin-tasks/src/main/java/org/apache/pinot/plugin/minion/tasks/upsertcompaction/UpsertCompactionTaskGenerator.java

```diff
@@ -124,9 +127,27 @@ public List<PinotTaskConfig> generateTasks(List<TableConfig> tableConfigs) {
 
       // TODO: currently, we put segmentNames=null to get metadata for all segments. We can change this to get
       // valid doc id metadata in batches with the loop.
+      String validDocIdType =
```

Comment:
nit: better to make this an enum

Reply:
@tibrewalpratik17 added enum. Can you double check?